crates/router/src/connector/klarna/transformers.rs

```diff
@@ -341,13 +543,35 @@ impl ForeignFrom<(KlarnaFraudStatus, bool)> for enums::AttemptStatus {
     }
 }
 
+impl From<KlarnaCheckoutStatus> for enums::AttemptStatus {
+    fn from(klarna_status: KlarnaCheckoutStatus) -> Self {
+        match klarna_status {
+            KlarnaCheckoutStatus::CheckoutComplete => Self::Charged,
```

Comment:
handle the manual capture flow here as it is handled in the above case

Reply:
@Sweta-Kumari-Sharma please address these comments, then only mark them as resolved.

Reply:
In from implementation, we can't have manual capture handling as klarna_checkout doesn't have specific status for manual capture flow. We just have "checkout_incomplete" status. The manual capture flow is handled in foreign_from implementation for checkout